src/site/markdown/getting-started.md

```diff
@@ -129,17 +129,24 @@ For the Context Class Loader case this would mean the following dependencies are
 
 ## Run
 
-Calling `executeGoals` runs Maven with the given goals or phases and optionally some additional environment variables. It throws a `VerificationException` in case the execution is not successful (e.g. binary not found or exit code > 0). It is either using a forked JVM or is executed in the same JVM depending on the configuration.
+Calling `execute` runs Maven with the given Verifier configuration like, cli arguments, environment variables. 
```

Comment:
```suggestion
Calling `execute` runs Maven with the given Verifier configuration like CLI arguments and environment variables. 
```

Reply:
The uppercase CLI should be everywhere in the document

Reply:
configuration, like ...

Reply:
CLI